File: 0232-implement-queue-using-stacks/0232-implement-queue-using-stacks.java
class MyQueue {
    ArrayList<Integer> que;

    public MyQueue() {
        que=new ArrayList<Integer>();
    }
    
    public void push(int x) {
       this.que.add(0,x);
    }
    
    public int pop() {
      return  this.que.remove(this.que.size()-1); 
    }
    
    public int peek() {
      return  this.que.get(this.que.size()-1); 
    }
    
    public boolean empty() {
      return this.que.size()==0;   
    }
}

/**
 * Your MyQueue object will be instantiated and called as such:
 * MyQueue obj = new MyQueue();
 * obj.push(x);
 * int param_2 = obj.pop();
 * int param_3 = obj.peek();
 * boolean param_4 = obj.empty();
 */

File: 0232-implement-queue-using-stacks/0232-implement-queue-using-stacks.py
class MyQueue:

    def __init__(self):
        self.que=[]
        

    def push(self, x: int) -> None:
        # print(x)
        self.que=[x]+self.que
        

    def pop(self) -> int:
       print(self.que)
       return self.que.pop(-1)
        

    def peek(self) -> int:
       return  self.que[-1]
        

    def empty(self) -> bool:
        return len(self.que)==0
        


# Your MyQueue object will be instantiated and called as such:
# obj = MyQueue()
# obj.push(x)
# param_2 = obj.pop()
# param_3 = obj.peek()
# param_4 = obj.empty()